src/openms/source/FORMAT/HANDLERS/MzMLHandler.cpp

```diff
@@ -1894,7 +1894,15 @@ namespace OpenMS::Internal
           {
             spec_.getPrecursors().back().getActivationMethods().insert(Precursor::SORI);
           }
-          else if (accession == "MS:1000422") //high-energy collision-induced dissociation
+          else if (accession == "MS:1000422") //beam-type collision-induced dissociation / HCD
+          {
+            spec_.getPrecursors().back().getActivationMethods().insert(Precursor::HCD);
+          }
+          else if (accession == "MS:1002472") //trap-type collision-induced dissociation
+          {
+            spec_.getPrecursors().back().getActivationMethods().insert(Precursor::TRAP);
+          }          
+          else if (accession == "MS:1002481") //high-energy collision-induced dissociation
```

Comment:
not really the fault of this PR, but this code duplication for spectra and chromatograms seems quite unneccessary... (e.g. grab a pointer to the respective precursor once, and write the insertion-mapping only once. However, totally optional :)